translations/translator.py

```diff
@@ -31,21 +50,55 @@ def parse_args(self, args, prog):
                             choices=("yaml", "json"),
                             default="yaml")
 
+        parser.add_argument('-d', '--dump',
+                            help='enable process to dump output files providing information about the execution',
+                            action='store_true',
+                            default=False)
+
         mode = parser.add_mutually_exclusive_group(required=True)
-        mode.add_argument('--generate',
-                          help='generate snapshot file',
-                          action='store_true', default=False)
+        mode.add_argument('-v', '--view',
+                          help='view translations state',
+                          action='store_true',
+                          default=False)
+        mode.add_argument('-e', '--export',
+                          help='generate export of pending translations',
+                          action='store_true',
+                          default=False)
+        mode.add_argument('-i', '--import-translations',
+                          dest='import_translations',
+                          help='import translations',
+                          action='store_true',
+                          default=False)
         mode.add_argument('--reconcile',
                           help='sanitize locale files',
-                          action='store_true', default=False)
+                          action='store_true',
+                          default=False)
         return parser.parse_args(args)
 
     def load_config(self):
-        if os.path.exists(config_file):
-            with open(config_file, 'r') as f:
-                data = yaml.full_load(f)
+        if not os.path.exists(config_file):
+            sys.exit(f'{self.whoami}: configuration file "{config_file}" not found')
+        with open(config_file, 'r') as f:
+            data = yaml.full_load(f)
         return data
 
+    def instantiate_exporter(self, config, options):
+        exporter_class_fqn = ConfigUtilities.get_value(config, ('io', 'out', 'generator'))
+        exporter_class = self.get_class(exporter_class_fqn)
+        return exporter_class(config, options)
+
+    def instantiate_importer(self, config, options):
+        importer_class_fqn = ConfigUtilities.get_value(config, ('io', 'in', 'importer'))
+        importer_class = self.get_class(importer_class_fqn)
+        return importer_class(config, options)
+
+    def get_class(self, class_fqn):
```

Comment:
There's other pythonic ways to get the class name? I don't quite know what the purpose of this is?

Reply:
I did not find any, but surely can look again.

Reply:
`__qualname__` gives you the class name for the current class for example. Is that what you were going for?